src/DotNet.Status.Web.Tests/AzurePipelinesControllerTests.cs

```diff
@@ -0,0 +1,350 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using DotNet.Status.Web.Options;
+using DotNet.Status.Web.Controllers;
+using Microsoft.DotNet.GitHub.Authentication;
+using Microsoft.DotNet.Internal.AzureDevOps;
+using Microsoft.DotNet.Internal.Testing.Utility;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using FluentAssertions;
+using Moq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+using Octokit;
+
+namespace DotNet.Status.Web.Tests
+{
+
+    [TestFixture]
+    public class AzurePipelinesControllerTests
+    {
+        public static string TestTeamsWebHookUri = "https://example.teams/webhook/sha";
```

Comment:
These should probably be const, or at least readonly. Going to have some really weird test behaviors if a test change ones of these, I bet you'll get surprising results. :-)

Reply:
I don't actually think I used that. That was a copy paste from one of Alex's tests. Will remove